### PR TITLE
Change password level

### PR DESCRIPTION
**Pull request by poupotte**

### CODE REVIEW

**Comment by gregorylegarec** [comment]
Thanks @poupotte 👍

File: test/lib/password_helper.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ describe('PasswordHelper', () => {
 
     it('should be between 33% and 66% "moderate" label for "Azerty1aze$"', () => {
       // arrange
-      let password = 'Azerty1aze$'
+      let password = 'Azerty1az'
```
Comment (lespacedunmatin) [inline test/lib/password_helper.js:33]: You have to change also the test title.

File: test/lib/password_helper.js [hunk 2]
```diff
@@ -43,7 +43,7 @@ describe('PasswordHelper', () => {
 
     it('should be above 66% and "strong" label for "Azerty1aze$uiopqs9LT"', () => {
       // arrange
-      let password = 'Azerty1aze$uiopqs9LT'
+      let password = 'Azerty1aze$uiopL'
```
Comment (lespacedunmatin) [inline test/lib/password_helper.js:46]: You have to change also the test title.

File: test/lib/password_helper.js (hunk 1 above)
Comment (gregorylegarec) [reply test/lib/password_helper.js:33]: Well spotted @lespacedunmatin  !

**Comment by gregorylegarec** [comment]
I quickfixed test titles in d6f41ce, thanks @lespacedunmatin  👍

**Pull request merged**